crates/bitwarden-core/src/auth/client_auth.rs

```diff
@@ -169,6 +165,7 @@ impl<'a> Client {
     }
 }
 
+/*
```

Comment:
This test is commented, should this be fixed or moved somewhere else?

Reply:
It's more of an integration test. I'm thinking we could move them to the bitwarden crate under a separate test directory.